### PR TITLE
[Tune] Fix docstring failures

### PR DESCRIPTION
**Pull request by justinvyu**
## Why are these changes needed?

This PR fixes the `Stopper` doctests that are erroring. Previously, it used a `tune.Trainable` as its trainable, which would error on fit since its methods are not implemented. Also, it was missing some imports.

## Checks

- [x] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: python/ray/tune/stopper/stopper.py [hunk 1]
```diff
@@ -15,34 +16,42 @@ class Stopper(abc.ABC):
...
-        ...     tune.Trainable,
-        ...     tune_config=tune.TuneConfig(num_samples=200),
-        ...     run_config=air.RunConfig(stop=TimeStopper())
+        ...     train_fn,
+        ...     tune_config=tune.TuneConfig(num_samples=2),
+        ...     run_config=air.RunConfig(stop=TimeStopper()),
         ... )
-        >>> tuner.fit()
-        == Status ==...
+        >>> print("[ignore]"); result_grid = tuner.fit()  # doctest: +ELLIPSIS
```
Comment (justinvyu) [inline python/ray/tune/stopper/stopper.py:43]: Any other way of doing this? This was the only workaround I could find :(

Comment (krfricke) [reply python/ray/tune/stopper/stopper.py:43]: I guess `lambda config: {"metric": 1}` or something would work, but also happy with this. LMK what you think

**Review by krfricke — APPROVED** [review]

**Comment by justinvyu** [comment]
@krfricke Looks like it's good to merge.

**Comment by cadedaniel** [comment]
This fixes broken doctests on the 2.3 release branch, let's merge this ASAP.

**Pull request merged**